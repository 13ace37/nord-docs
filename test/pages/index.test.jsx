/*
 * Copyright (C) 2018-present Arctic Ice Studio <development@arcticicestudio.com>
 * Copyright (C) 2018-present Sven Greb <development@svengreb.de>
 *
 * Project:    Nord Docs
 * Repository: https://github.com/arcticicestudio/nord-docs
 * License:    MIT
 */

import React from "react";
import { render } from "react-testing-library";

import Landing from "pages";

/*
 * This test is disabled because it will be covered by E2E tests with Cypress later on.
 * It is also skipped until due to problems with mocking transitive modules and components.
 */
test.skip("snapshot", () => {
  const { container } = render(<Landing />);
  expect(container).toMatchSnapshot();
});
