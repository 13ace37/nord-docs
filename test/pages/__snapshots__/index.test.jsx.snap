// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <main
    class="sc-bdVaJa eYFyfX"
  />
</div>
`;
